x-pack/metricbeat/module/aws/utils.go

```diff
@@ -67,12 +67,12 @@ func GetListMetricsOutput(namespace string, regionName string, svcCloudwatch clo
 
 // GetMetricDataResults function uses MetricDataQueries to get metric data output.
 func GetMetricDataResults(metricDataQueries []types.MetricDataQuery, svc cloudwatch.GetMetricDataAPIClient, startTime time.Time, endTime time.Time) ([]types.MetricDataResult, error) {
-	maxQuerySize := 100
+	maxQuerySize := 500
```

Comment:
NIT: I think it'd be better to change the name of this variable to something like ```maxNumberOfQueries``` or something similar, just to make clear that it contains the maximum number of queries that can be passed in a single ```getMetricData``` API call. ```maxQuerySize``` may be confusing for a reader without context.

Just an observation. Not at all blocking this PR.

Reply:
Sounds good! Thanks you! I also just realized this is actually `maxNumberOfMetricsRetrieved` in a single query. So I will change it to `maxNumberOfMetricsRetrieved` instead.